get_item_ranges.py

```
@@ -0,0 +1,50 @@
+import re
+import json
+
+def get_ranges():
+    with open('/tmp/telemetry.rs', 'r') as f:
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/telemetry.rs"

(S108)